packages/devextreme/js/__internal/ui/chat/avatar.ts

```diff
@@ -99,14 +99,25 @@ class Avatar extends Widget<Properties> {
     return result;
   }
 
-  _getInitials(name: string | undefined): string {
-    if (isDefined(name)) {
-      return String(name).charAt(0).toUpperCase();
+  _getInitials(name: string | number | undefined): string {
+    if (isString(name) || isNumeric(name)) {
+      const splitValue = String(name).trim().split(' ');
+
+      const firstInitial = this._getFirstChar(splitValue[0]);
+      const secondInitial = this._getFirstChar(splitValue[1]);
+
+      const result = `${firstInitial}${secondInitial}`;
+
+      return result;
     }
 
     return '';
```

Comment:
Did you consider variant when we return some predefined value, not empty string? For example, it can be 'UU' (Unknown User) 
<img width="50" alt="image" src="https://github.com/user-attachments/assets/00bd8ed3-2273-4a1e-8306-14e7d29d7d99">

Reply:
Let's make a card for researching this question. Because the scenario you described has a broader meaning. Maybe we should set a default value for User (just like for id). Because having only a default value for initials is weird, because it might confuse users.

Reply:
Moved to: https://github.com/DevExpress/DevExtreme/pull/28062#discussion_r1763318640